applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.h

```diff
@@ -41,6 +41,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) CoulombYieldSurface : public YieldSu
     [[nodiscard]] double GetFrictionAngleInRadians() const;
     [[nodiscard]] double GetCohesion() const;
     [[nodiscard]] double GetDilatancyAngleInRadians() const;
+    [[nodiscard]] double GetKappa() const;
+
+    void SetKappa(double kappa);
```

Comment:
I would put these two closer together, since they are about the same parameter (i.e. $`\kappa`$).
```suggestion

    [[nodiscard]] double GetKappa() const;
    void SetKappa(double kappa);
```